src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,146 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { MdArrowForwardIos } from 'react-icons/md';
```

Comment:
I think here also you need to find the correspondent under 'react-icons/fi'.

Reply:
Here: https://react-icons.github.io/react-icons/icons?name=fi

Reply:
yes it's "import { FiChevronRight } from 'react-icons/fi';"